packages/cms/src/components/editors/vizLookup.js

```diff
@@ -0,0 +1,138 @@
+// Placeholder for any future universal keys
+const sharedMethods = [];
+
+// Create a lookup for some commonly used keys
+const commonKeys = ["groupBy", "x", "y", "value", "colorScale", "label", "subtitle", "sum"];
+const commonMethods = {};
+commonKeys.forEach(key => {
+  commonMethods[key] = {
+    key,
+    display: key,
+    format: "Accessor",
+    required: true
+  };
+});
+// Override some of the labels and types
+commonMethods.groupBy.typeof = "id";
+commonMethods.groupBy.display = "Grouping";
+commonMethods.colorScale.display = "Color Scale";
+commonMethods.value.typeof = "number";
+commonMethods.sum.typeof = "number";
+
+// Create a lookup for some optional keys. Nested properties are handled by dots, e.g. yConfig.title
+const optionalKeys = ["xConfig", "yConfig"];
+optionalKeys.forEach(key => {
+  commonMethods[key] = {
+    key: `${key}.title`,
+    display: `${key.charAt(0).toUpperCase()} Axis Label`,
+    format: "Input",
+    required: false
+  };
+});
+
+// Promote certain keys to appear in the tooltip by default
+const tooltipKeys = ["x", "y", "value", "sum"];
+tooltipKeys.forEach(key => {
+  commonMethods[key].tooltip = true;
+});
+const {groupBy, x, y, value, colorScale, label, subtitle, sum, xConfig, yConfig} = commonMethods;
```

Comment:
1 & 2. Sounds good.

3. Title was originally in the now-removed `sharedMethods`. Title is kind of its "own thing" in that it is formatted to be outside of the normal keylisting:

![image](https://user-images.githubusercontent.com/1714292/68502372-3f2ebc00-022e-11ea-936d-bb05987c24e3.png)

Indeed, I'm excluding title from the schema config, because all I would do on the Front-end is just rip it out anyway and put it in its special non-looped place (and also filter the title OUT of the methods in every other place the keys get listed). I am of the opinion that global keys should, design-wise and schema-wise, stand alone visually in the SimpleVizConfig.  The vizLookup is for the "iterables" that are unique to any given viz.

4. They are called `xConfig.title` and `yConfig.title` right now, I use a template string.